extra-tests/classes/InvocableDrivenTests.cls

```diff
@@ -9,8 +9,28 @@ private class InvocableDrivenTests {
     insert acc;
   }
 
+  @IsTest
+  static void shouldWorkWithRefreshContext() {
+    Account acc = [SELECT Id FROM Account];
+
+    // Description and Subject both are referenced in the Flow
```

Comment:
I have a comment at the top of this test class about the dependency on the Flow - because there weren't any changes to that section of it, it's probably not easy to see that, but it's ~10 lines of code from the class declaration (and that comment about the flow dependency) to this line, so I think I'm going to keep this one as is.